### PR TITLE
On circleCI do not run buildModules twice

### PR DESCRIPTION
**Pull request by alisman**
We already run `yarn build` which includes `buildModules`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,7 +89,6 @@ jobs:
           - v2-dependencies-{{ checksum "yarn.lock" }}
       # Download and cache dependencies
       - run: yarn
-      - run: yarn buildModules
```
Comment (inodb) [inline .circleci/config.yml]: @alisman where else is it run? Can't seem to find it. Is it run as part of some other yarn job?

Comment (onursumer) [reply .circleci/config.yml]: @inodb `yarn build` runs `yarn buildModules` as of the latest setup

Comment (inodb) [reply .circleci/config.yml]: gotcha - thanks!

**Comment by alisman** [comment]
@onursumer @inodb seems that this may be necessary after all for unit test run.  anyhow, it doesn't actually seem to improve Circle CI build time, so gonna close it.

**Pull request closed without merging**